pkg/azure-storage/AzureStorageConsumer.php

```diff
@@ -91,8 +91,7 @@ public function reject(Message $message, bool $requeue = false): void
         InvalidMessageException::assertMessageInstanceOf($message, AzureStorageMessage::class);
 
         if (true === $requeue) {
-            $factory = new AzureStorageConnectionFactory($this->client);
-            $context = $factory->getContext();
+            $context = new AzureStorageContext($this->client);
```

Comment:
I think context should be passed in consumer constructor. The context should have a getClient public method